features/opennms-es-rest/src/main/java/org/opennms/plugins/elasticsearch/rest/EventToIndex.java

```diff
@@ -97,6 +97,8 @@
 	private int threads = DEFAULT_NUMBER_OF_THREADS;
 
 	private IndexStrategy indexStrategy = IndexStrategy.MONTHLY;
+	
+	private String eventIndexName = "opennms";
```

Comment:
Again, here we should probably use empty string "" as the default. Also we will need a setter method for this value in this class since it is being set via property in the blueprint.